src/libraries/Microsoft.PowerFx.LanguageServerProtocol/Protocol/PublishControlTokensParams.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.PowerFx.Intellisense;
+using Microsoft.PowerFx.LanguageServerProtocol.Schemas;
+
+namespace Microsoft.PowerFx.LanguageServerProtocol.Protocol
+{
+    internal class PublishControlTokensParams
+    {
+        /// <summary>
+        /// The Version for which token information is reported.
+        /// </summary>
+        public string Version { get; set; } = string.Empty;
```

Comment:
Not for now but in future we should send back the whole uri once we move to workspace architecture in the new formula bar.